### PR TITLE
Avoid going to the database to compute a value that we already have in memory.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @sharwell

**Comment by CyrusNajmabadi** [comment]
> What happens in ReadBlob if the data doesn't already exist?

Good point.  Have updated.

File: src/Workspaces/Core/Portable/Shared/Extensions/SyntaxTriviaListExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Shared.Extensions
+{
+    internal static class SyntaxTriviaListExtensions
+    {
+       public static SyntaxTrivia LastOrDefault(this SyntaxTriviaList triviaList)
+            => triviaList.Count == 0 ? default(SyntaxTrivia) : triviaList.Last();
```
Comment (sharwell) [inline src/Workspaces/Core/Portable/Shared/Extensions/SyntaxTriviaListExtensions.cs]: :bulb: `default`

File: src/Workspaces/Core/Portable/Workspaces.csproj [hunk 2]
```diff
@@ -213,6 +213,10 @@
     <Compile Include="Formatting\IFormattingService.cs" />
     <Compile Include="Remote\RemoteHostSessionHelpers.cs" />
     <Compile Include="Remote\RemoteFeatureOptions.cs" />
+<<<<<<< HEAD
```
Comment (sharwell) [inline src/Workspaces/Core/Portable/Workspaces.csproj]: ❕ Merge conflict

**Comment by sharwell** [comment]
@CyrusNajmabadi Can you rebase this and remove the changes to unrelated files? It's a single small change so it shouldn't have 14 commits.

**Pull request merged**